universe/Condition.cpp

```diff
@@ -7511,7 +7511,7 @@ namespace {
         dy2 /= mag;
 
 
-        const float MAX_LANE_DOT_PRODUCT = 0.98f;   // magic limit copied from CullAngularlyTooCloseLanes in UniverseGenerator
+        const float MAX_LANE_DOT_PRODUCT = 0.87f;   // magic limit copied from CullAngularlyTooCloseLanes in UniverseGenerator
```

Comment:
update comment to remove reference to https://github.com/freeorion/freeorion/blob/master/universe/UniverseGenerator.cpp#L743

Reply:
Hopefully fixed